docs/framework/guides/deep-dive/localization.md

```diff
@@ -119,12 +125,15 @@ editor.ui.componentFactory.add( 'smilingFaceEmoji', locale => {
 
 ```js
 class FileRepository {
+	// Other class properties.
 	// ...
+	
 	updatePendingAction() {
-		// ...
 		const pendingActions = this.editor.plugins.get( PendingActions );
 
-		// ...
+		// Declarations of other variables and functions.
```

Comment:
Actually, this is placed within a function, so other functions cannot be defined here. I consulted with @pomek and we think that removing those 2 ellipses within the `updatePendingAction()` will be sufficient in this case. :)

Reply:
Great, thanks! I've removed the ellipses here. :blush:

Reply:
Thanks! :) However, we could leave the ellipse at the beginning of the `FileRepository` branch with a comment like "More methods." or something similar :)

Reply:
Sure thing. It's done now :)